tests/test_dataset_v2.py

```diff
@@ -65,6 +65,13 @@ def test_dataset_v2_load_to_memory(test_dataset_v2):
     assert d2 < d1
 
 
+def test_dataset_v2_checksum(test_dataset_v2):
+    # Make sure the `md5sum` is part of the model dump even if not initiated yet.
+    # This is important for uploads to the Hub.
+    assert test_dataset_v2._md5sum is None
+    assert "md5sum" in test_dataset_v2.model_dump()
```

Comment:
**Comment:** Since the `_md5sum` property shouldn't be part of V2, this test case doesn't make sense.